deploy/crds/apps.3scale.net_apimanagers_crd.yaml

```diff
@@ -107,13 +119,34 @@ spec:
                   type: object
                 database:
                   properties:
+                    embedded:
+                      properties:
+                        mysql:
+                          properties:
+                            image:
+                              type: string
+                          type: object
+                        postgresql:
+                          properties:
+                            image:
+                              type: string
+                          type: object
+                      type: object
+                    external:
```

Comment:
in 'external' section I've added the mysql and postgresql section but I'm not sure whether we should differentiate in this case. In any case I've added them but I'm not using them in the code at the moment (in code I just check whether external is nil or not)